{"version": 2, "width": 110, "height": 23, "timestamp": 1540349916, "idle_time_limit": 4.0, "env": {"SHELL": "/bin/bash", "TERM": "xterm-256color"}}
[0.022908, "o", "\u001b]0;jcrigby@Hoid: /tmp/tmp6ek4qu75\u0007jcrigby@Hoid:/tmp/tmp6ek4qu75$ (master) "]
[0.855314, "o", " export PS1=\"# \""]
[0.856992, "o", "\r\n"]
[0.857102, "o", "# "]
[0.858928, "o", " export LESS=\"-M -I -R -S\""]
[0.860861, "o", "\r\n"]
[0.861022, "o", "# "]
[0.862583, "o", "\r\n"]
[0.862651, "o", "# "]
[0.864494, "o", "\r\n# "]
[0.866429, "o", "\r\n# "]
[0.868113, "o", "\r\n# "]
[0.869743, "o", "\r\n"]
[0.869819, "o", "# "]
[0.871361, "o", "\r\n"]
[0.871438, "o", "# "]
[0.873008, "o", "\r\n"]
[0.873071, "o", "# "]
[0.874831, "o", "\r\n# "]
[0.876358, "o", "\r\n"]
[0.876421, "o", "# "]
[0.878172, "o", "\r\n"]
[0.878258, "o", "# "]
[0.879819, "o", "\r\n"]
[0.879886, "o", "# "]
[0.881501, "o", "\r\n# "]
[0.883259, "o", "\r\n# "]
[0.88487, "o", "\r\n# "]
[0.886525, "o", "\r\n# "]
[0.888133, "o", "\r\n# "]
[0.889801, "o", "\r\n# "]
[0.891467, "o", "\r\n# "]
[0.893005, "o", "\r\n# "]
[0.894727, "o", "\r\n# "]
[0.896276, "o", "\r\n"]
[0.89636, "o", "# "]
[0.898421, "o", "\r\n# "]
[0.900185, "o", "\r\n# "]
[0.901941, "o", "\r\n# "]
[0.903627, "o", "\r\n# "]
[0.905159, "o", "\r\n# "]
[0.906895, "o", "\r\n# "]
[0.908648, "o", "\r\n# "]
[0.910313, "o", "\r\n# "]
[0.91195, "o", "\r\n# "]
[0.913619, "o", "\r\n# "]
[0.915306, "o", "\r\n# "]
[0.916836, "o", "\r\n# "]
[0.918496, "o", "\r\n# "]
[0.920166, "o", "\r\n# "]
[0.921863, "o", "\r\n# "]
[0.923456, "o", "\r\n# "]
[0.924994, "o", "\r\n# "]
[0.926609, "o", "\r\n# "]
[0.928365, "o", "\r\n# "]
[4.979633, "o", "git remote update"]
[4.985616, "o", "\r\n"]
[4.991468, "o", "Fetching origin\r\n"]
[5.006552, "o", "remote: Counting objects: 3, done.\u001b[K\r\n"]
[5.031695, "o", "remote: Total 3 (delta 0), reused 0 (delta 0)\u001b[K\r\n"]
[5.03518, "o", "Unpacking objects:  33% (1/3)   \r"]
[5.035543, "o", "Unpacking objects:  66% (2/3)   \r"]
[5.035828, "o", "Unpacking objects: 100% (3/3)   \rUnpacking objects: 100% (3/3), done.\r\n"]
[5.041123, "o", "From /tmp/tmpczzekvih\r\n * [new branch]      master     -> origin/master\r\n"]
[5.048713, "o", "# "]
[5.994247, "o", "git checkout master"]
[5.999691, "o", "\r\n"]
[6.029299, "o", "Branch 'master' set up to track remote branch 'master' from 'origin'.\r\n"]
[6.029832, "o", "Already on 'master'\r\n"]
[6.030588, "o", "# "]
[12.010601, "o", "echo foo > file2.txt"]
[12.014088, "o", "\r\n"]
[12.014264, "o", "# "]
[13.021889, "o", "git add file2.txt"]
[13.027845, "o", "\r\n"]
[13.031908, "o", "# "]
[14.036459, "o", "git commit -m \"added to file2.txt\""]
[14.042664, "o", "\r\n"]
[14.051807, "o", "[master af2b412] added to file2.txt\r\n"]
[14.051907, "o", " 1 file changed, 1 insertion(+)\r\n create mode 100644 file2.txt\r\n"]
[14.052504, "o", "# "]
[19.108948, "o", "git push"]
[19.115399, "o", "\r\n"]
[19.12372, "o", "To /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\n"]
[19.124076, "o", "error: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n"]
[19.124655, "o", "# "]
[20.130183, "o", "\r\n"]
[20.190548, "o", "\u001b[?1049h\u001b[22;0;0t"]
[20.191715, "o", "Unpacking objects: 100% (3/3), done.\r\nFrom /tmp/tmpczzekvih\r\n * [new branch]      master     -> origin/master\r\n# git checkout master\r\nBranch 'master' set up to track remote branch 'master' from 'origin'.\r\nAlready on 'master'\r\n# echo foo > file2.txt\r\n# git add file2.txt\r\n# git commit -m \"added to file2.txt\"\r\n[master af2b412] added to file2.txt\r\n 1 file changed, 1 insertion(+)\r\n create mode 100644 file2.txt\r\n# git push\r\nTo /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\nerror: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n"]
[20.192941, "o", "\u001b[31m"]
[20.193449, "o", "\u001b[1m"]
[20.19582, "o", "\u001b[2;73H"]
[20.196164, "o", "╭──────────────────────────────────╮\r\n"]
[20.197185, "o", "\u001b[3;73H"]
[20.197323, "o", "│ push failed                      │\r\n"]
[20.198272, "o", "\u001b[4;73H"]
[20.198589, "o", "│ git pull --rebase and try again  │\r\n"]
[20.199636, "o", "\u001b[5;73H"]
[20.199791, "o", "╰──────────────────────────────────╯\r\n"]
[20.200829, "o", "\u001b[6;73H"]
[20.201131, "o", "\r\n"]
[20.20208, "o", "\u001b(B\u001b[m"]
[20.203227, "o", "\u001b[1000;1H"]
[25.212693, "o", "\u001b[?1049l\u001b[23;0;0t"]
[25.217158, "o", "\u001b[1000;1H"]
[25.231254, "o", "# "]
[26.143569, "o", "git pull --rebase"]
[26.149301, "o", "\r\n"]
[26.171756, "o", "remote: Counting objects: 3, done.\u001b[K\r\n"]
[26.172434, "o", "remote: Total 3 (delta 0), reused 0 (delta 0)\u001b[K\r\n"]
[26.175824, "o", "Unpacking objects:  33% (1/3)   \r"]
[26.17617, "o", "Unpacking objects:  66% (2/3)   \r"]
[26.176504, "o", "Unpacking objects: 100% (3/3)   \rUnpacking objects: 100% (3/3), done.\r\n"]
[26.181765, "o", "From /tmp/tmpczzekvih\r\n   520376f..b5b02bc  master     -> origin/master\r\n"]
[26.258762, "o", "First, rewinding head to replay your work on top of it...\r\n"]
[26.269417, "o", "Applying: added to file2.txt\r\n"]
[26.278037, "o", "# "]
[31.215167, "o", "git push"]
[31.221534, "o", "\r\n"]
[31.229873, "o", "To /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\n"]
[31.230166, "o", "error: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n"]
[31.230685, "o", "# "]
[32.236276, "o", "\r\n"]
[32.290467, "o", "\u001b[?1049h\u001b[22;0;0t"]
[32.291631, "o", "hint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n# git pull --rebase\r\nremote: Counting objects: 3, done.\r\nremote: Total 3 (delta 0), reused 0 (delta 0)\r\nUnpacking objects: 100% (3/3), done.\r\nFrom /tmp/tmpczzekvih\r\n   520376f..b5b02bc  master     -> origin/master\r\nFirst, rewinding head to replay your work on top of it...\r\nApplying: added to file2.txt\r\n# git push\r\nTo /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\nerror: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n"]
[32.292883, "o", "\u001b[31m"]
[32.293396, "o", "\u001b[1m"]
[32.295728, "o", "\u001b[2;73H"]
[32.295955, "o", "╭──────────────────────────────────╮\r\n"]
[32.297017, "o", "\u001b[3;73H"]
[32.297239, "o", "│ push failed again                │\r\n"]
[32.298173, "o", "\u001b[4;73H"]
[32.298316, "o", "│ git pull --rebase and try again  │\r\n"]
[32.299292, "o", "\u001b[5;73H"]
[32.299595, "o", "╰──────────────────────────────────╯\r\n"]
[32.300652, "o", "\u001b[6;73H"]
[32.300919, "o", "\r\n"]
[32.301939, "o", "\u001b(B\u001b[m"]
[32.303061, "o", "\u001b[1000;1H"]
[37.312635, "o", "\u001b[?1049l\u001b[23;0;0t"]
[37.31701, "o", "\u001b[1000;1H"]
[37.33193, "o", "# "]
[38.249701, "o", "git pull --rebase"]
[38.256062, "o", "\r\n"]
[38.279665, "o", "remote: Counting objects: 3, done.\u001b[K\r\n"]
[38.280263, "o", "remote: Total 3 (delta 0), reused 0 (delta 0)\u001b[K\r\n"]
[38.283693, "o", "Unpacking objects:  33% (1/3)   \r"]
[38.284061, "o", "Unpacking objects:  66% (2/3)   \r"]
[38.284443, "o", "Unpacking objects: 100% (3/3)   \rUnpacking objects: 100% (3/3), done.\r\n"]
[38.289846, "o", "From /tmp/tmpczzekvih\r\n   b5b02bc..5a1ffab  master     -> origin/master\r\n"]
[38.369017, "o", "First, rewinding head to replay your work on top of it...\r\n"]
[38.390772, "o", "Applying: added to file2.txt\r\n"]
[38.414642, "o", "# "]
[43.322821, "o", "git push"]
[43.329037, "o", "\r\n"]
[43.337579, "o", "To /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\n"]
[43.337881, "o", "error: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n"]
[43.338489, "o", "# "]
[44.342606, "o", "\r\n"]
[44.400308, "o", "\u001b[?1049h\u001b[22;0;0t"]
[44.401518, "o", "hint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n# git pull --rebase\r\nremote: Counting objects: 3, done.\r\nremote: Total 3 (delta 0), reused 0 (delta 0)\r\nUnpacking objects: 100% (3/3), done.\r\nFrom /tmp/tmpczzekvih\r\n   b5b02bc..5a1ffab  master     -> origin/master\r\nFirst, rewinding head to replay your work on top of it...\r\nApplying: added to file2.txt\r\n# git push\r\nTo /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\nerror: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n"]
[44.402698, "o", "\u001b[31m"]
[44.403179, "o", "\u001b[1m"]
[44.405425, "o", "\u001b[2;73H"]
[44.405724, "o", "╭──────────────────────────────────╮\r\n"]
[44.406801, "o", "\u001b[3;73H"]
[44.407002, "o", "│ push failed again                │\r\n"]
[44.408152, "o", "\u001b[4;73H"]
[44.408364, "o", "│ git pull --rebase and try again  │\r\n"]
[44.409346, "o", "\u001b[5;73H"]
[44.409603, "o", "╰──────────────────────────────────╯\r\n"]
[44.410717, "o", "\u001b[6;73H"]
[44.41094, "o", "\r\n"]
[44.411927, "o", "\u001b(B\u001b[m"]
[44.413011, "o", "\u001b[1000;1H"]
[49.417085, "o", "\u001b[?1049l\u001b[23;0;0t"]
[49.419695, "o", "\u001b[1000;1H"]
[49.427186, "o", "# "]
[50.355111, "o", "git pull --rebase"]
[50.361434, "o", "\r\n"]
[50.385878, "o", "remote: Counting objects: 3, done.\u001b[K\r\n"]
[50.386619, "o", "remote: Total 3 (delta 0), reused 0 (delta 0)\u001b[K\r\n"]
[50.390087, "o", "Unpacking objects:  33% (1/3)   \r"]
[50.390482, "o", "Unpacking objects:  66% (2/3)   \r"]
[50.390823, "o", "Unpacking objects: 100% (3/3)   \rUnpacking objects: 100% (3/3), done.\r\n"]
[50.396238, "o", "From /tmp/tmpczzekvih\r\n   5a1ffab..2db2a5a  master     -> origin/master\r\n"]
[50.47121, "o", "First, rewinding head to replay your work on top of it...\r\n"]
[50.480829, "o", "Applying: added to file2.txt\r\n"]
[50.489132, "o", "# "]
[55.388534, "o", "git push"]
[55.390402, "o", "\r\n"]
[55.393213, "o", "To /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\n"]
[55.39325, "o", "error: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\n"]
[55.393314, "o", "hint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n"]
[55.393526, "o", "# "]
[56.395567, "o", "\r\n"]
[56.421528, "o", "\u001b[?1049h\u001b[22;0;0t"]
[56.422629, "o", "hint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n# git pull --rebase\r\nremote: Counting objects: 3, done.\r\nremote: Total 3 (delta 0), reused 0 (delta 0)\r\nUnpacking objects: 100% (3/3), done.\r\nFrom /tmp/tmpczzekvih\r\n   5a1ffab..2db2a5a  master     -> origin/master\r\nFirst, rewinding head to replay your work on top of it...\r\nApplying: added to file2.txt\r\n# git push\r\nTo /tmp/tmpczzekvih\r\n ! [rejected]        master -> master (fetch first)\r\nerror: failed to push some refs to '/tmp/tmpczzekvih'\r\nhint: Updates were rejected because the remote contains work that you do\r\nhint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n"]
[56.423726, "o", "\u001b[31m"]
[56.424182, "o", "\u001b[1m"]
[56.426453, "o", "\u001b[2;73H"]
[56.426634, "o", "╭──────────────────────────────────╮\r\n"]
[56.427598, "o", "\u001b[3;73H"]
[56.42779, "o", "│ push failed again                │\r\n"]
[56.428827, "o", "\u001b[4;73H"]
[56.429003, "o", "│ git pull --rebase and try again  │\r\n"]
[56.430047, "o", "\u001b[5;73H"]
[56.43017, "o", "╰──────────────────────────────────╯\r\n"]
[56.431244, "o", "\u001b[6;73H"]
[56.431386, "o", "\r\n"]
[56.432369, "o", "\u001b(B\u001b[m"]
[56.433491, "o", "\u001b[1000;1H"]
[61.440046, "o", "\u001b[?1049l\u001b[23;0;0t"]
[61.444408, "o", "\u001b[1000;1H"]
[61.458418, "o", "# "]
[62.405003, "o", "git pull --rebase"]
[62.407481, "o", "\r\n"]
[62.420934, "o", "remote: Counting objects: 3, done.\u001b[K\r\n"]
[62.421324, "o", "remote: Total 3 (delta 0), reused 0 (delta 0)\u001b[K\r\n"]
[62.424359, "o", "Unpacking objects:  33% (1/3)   \r"]
[62.424692, "o", "Unpacking objects:  66% (2/3)   \r"]
[62.425029, "o", "Unpacking objects: 100% (3/3)   \rUnpacking objects: 100% (3/3), done.\r\n"]
[62.430399, "o", "From /tmp/tmpczzekvih\r\n   2db2a5a..c5ddd7f  master     -> origin/master\r\n"]
[62.507611, "o", "First, rewinding head to replay your work on top of it...\r\n"]
[62.518287, "o", "Applying: added to file2.txt\r\n"]
[62.526819, "o", "# "]
[63.415119, "o", "git push"]
[63.421447, "o", "\r\n"]
[63.433075, "o", "Counting objects: 3, done.\r\n"]
[63.433334, "o", "Delta compression using up to 8 threads.\r\nCompressing objects:  50% (1/2)   \rCompressing objects: 100% (2/2)   \r"]
[63.433698, "o", "Compressing objects: 100% (2/2), done.\r\n"]
[63.434851, "o", "Writing objects:  33% (1/3)   \rWriting objects:  66% (2/3)   \rWriting objects: 100% (3/3)   \rWriting objects: 100% (3/3), 294 bytes | 294.00 KiB/s, done.\r\nTotal 3 (delta 0), reused 0 (delta 0)\r\n"]
[63.448978, "o", "To /tmp/tmpczzekvih\r\n   c5ddd7f..1f697ea  master -> master\r\n"]
[63.450145, "o", "# "]
[64.430798, "o", "\r\n"]
[64.456276, "o", "\u001b[?1049h\u001b[22;0;0t"]
[64.45735, "o", "hint: not have locally. This is usually caused by another repository pushing\r\nhint: to the same ref. You may want to first integrate the remote changes\r\nhint: (e.g., 'git pull ...') before pushing again.\r\nhint: See the 'Note about fast-forwards' in 'git push --help' for details.\r\n# \r\n# git pull --rebase\r\nremote: Counting objects: 3, done.\r\nremote: Total 3 (delta 0), reused 0 (delta 0)\r\nUnpacking objects: 100% (3/3), done.\r\nFrom /tmp/tmpczzekvih\r\n   2db2a5a..c5ddd7f  master     -> origin/master\r\nFirst, rewinding head to replay your work on top of it...\r\nApplying: added to file2.txt\r\n# git push\r\nCounting objects: 3, done.\r\nDelta compression using up to 8 threads.\r\nCompressing objects: 100% (2/2), done.\r\nWriting objects: 100% (3/3), 294 bytes | 294.00 KiB/s, done.\r\nTotal 3 (delta 0), reused 0 (delta 0)\r\nTo /tmp/tmpczzekvih\r\n   c5ddd7f..1f697ea  master -> master\r\n# \r\n"]
[64.458511, "o", "\u001b[31m"]
[64.458958, "o", "\u001b[1m"]
[64.461409, "o", "\u001b[2;90H"]
[64.461601, "o", "╭─────────────────╮\r\n"]
[64.462525, "o", "\u001b[3;90H"]
[64.46269, "o", "│ push succeeded! │\r\n"]
[64.463598, "o", "\u001b[4;90H"]
[64.46377, "o", "╰─────────────────╯\r\n"]
[64.464735, "o", "\u001b[5;90H"]
[64.464839, "o", "\r\n"]
[64.465817, "o", "\u001b(B\u001b[m"]
[64.467006, "o", "\u001b[1000;1H"]
[65.438439, "o", "git lg3"]
[65.444783, "o", "\r\n"]
[69.476477, "o", "\u001b[?1049l\u001b[23;0;0t"]
[69.480848, "o", "\u001b[1000;1H"]
[69.487568, "o", "# git lg3\r\n"]
[69.491513, "o", "\u001b[?1049h\u001b[22;0;0t\u001b[?1h\u001b=\r"]
[69.491649, "o", "* \u001b[1;34m1f697ea\u001b[m - \u001b[1;36mTue, 23 Oct 2018 20:58:50 -0600\u001b[m \u001b[1;32m(56 seconds ago)\u001b[m \u001b[1;36m(committed: Tue, 23 Oct 2018 20:59:39 -0600)\u001b[m \u001b[1;33m (HE\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[37madded to file2.txt\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[2;37m- SpongeBob SquarePants <spongebob@nick.com> \u001b[m \u001b[2;37m(committer: SpongeBob SquarePants <spongebob@nick.co\u001b[m\r\n* \u001b[1;34mc5ddd7f\u001b[m - \u001b[1;36mTue, 23 Oct 2018 20:59:30 -0600\u001b[m \u001b[1;32m(16 seconds ago)\u001b[m \u001b[1;36m(committed: Tue, 23 Oct 2018 20:59:30 -0600)\u001b[m \u001b[1;33m\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[37mappended to file1.txt\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[2;37m- Patrick Star <patrick@nick.com> \u001b[m \u001b[2;37m(committer: Patrick Star <patrick@nick.com>)\u001b[m\u001b[m\r\n* \u001b[1;34m2db2a5a\u001b[m - \u001b[1;36mTue, 23 Oct 2018 20:59:18 -0600\u001b[m \u001b[1;32m(28 seconds ago)\u001b[m \u001b[1;36m(committed: Tue, 23 Oct 2018 20:59:18 -0600)\u001b[m \u001b[1;33m\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[37mappended to file1.txt\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[2;37m- Patrick Star <patrick@nick.com> \u001b[m \u001b[2;37m(committer: Patrick Star <patrick@nick.com>)\u001b"]
[69.491748, "o", "[m\u001b[m\r\n* \u001b[1;34m5a1ffab\u001b[m - \u001b[1;36mTue, 23 Oct 2018 20:59:05 -0600\u001b[m \u001b[1;32m(41 seconds ago)\u001b[m \u001b[1;36m(committed: Tue, 23 Oct 2018 20:59:05 -0600)\u001b[m \u001b[1;33m\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[37mappended to file1.txt\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[2;37m- Patrick Star <patrick@nick.com> \u001b[m \u001b[2;37m(committer: Patrick Star <patrick@nick.com>)\u001b[m\u001b[m\r\n* \u001b[1;34mb5b02bc\u001b[m - \u001b[1;36mTue, 23 Oct 2018 20:58:53 -0600\u001b[m \u001b[1;32m(53 seconds ago)\u001b[m \u001b[1;36m(committed: Tue, 23 Oct 2018 20:58:53 -0600)\u001b[m \u001b[1;33m\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[37mappended to file1.txt\u001b[m\u001b[m\r\n\u001b[31m|\u001b[m           \u001b[2;37m- Patrick Star <patrick@nick.com> \u001b[m \u001b[2;37m(committer: Patrick Star <patrick@nick.com>)\u001b[m\u001b[m\r\n* \u001b[1;34m520376f\u001b[m - \u001b[1;36mTue, 23 Oct 2018 20:58:39 -0600\u001b[m \u001b[1;32m(67 seconds ago)\u001b[m \u001b[1;36m(committed: Tue, 23 Oct 2018 20:58:39 -0600)\u001b[m \u001b[1;33m\u001b[m\u001b[m\r\n            \u001b[37madded file1.txt\u001b[m\u001b[m\r\n            \u001b[2;37m- Patrick Star <patrick@nick.com> \u001b[m \u001b[2;37m(committer: Patrick Star <patrick@nick.com>)\u001b[m\u001b[m\r\n\u001b[7mlines 1-18/18 (END"]
